force-app/test/utilities/test-environment.js

```diff
@@ -28,6 +28,9 @@
  * When creating an instance of this class, pass the environment name prefix as a parameter to the constructor
  * (i.e new TestEnvironment("sandbox"))
  */
+
+const SESSION_TIMEOUT = 2 * 60 * 1000; // 2 hours by default
```

Comment:
Is this the default timeout for scratch orgs created with SFDX CLI?

Reply:
Yes, this is the default timeout.

<img width="724" alt="Screen Shot 2022-05-21 at 08 35 06" src="https://user-images.githubusercontent.com/5071767/169639257-baf38a38-ca0a-4454-b367-6aa76201a65e.png">